app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -215,6 +215,7 @@ pageNofMText=Page {{n}} of {{m}}
 paintTabs=Show tabs in page theme color
 passwordManager=Password Manager
 passwordsAndForms=Passwords and Forms
+paymentsAllowPromotions=Show BAT promotions
```

Comment:
Per comment from @davidtemkin, I believe this should be "token" promotions

Reply:
done

Reply:
yes, thanks